src/shapes/IText/ITextBehavior.ts

```diff
@@ -160,7 +160,7 @@ export abstract class ITextBehavior<
   private _onTickComplete() {
     this._currentTickCompleteState?.abort();
     this._currentTickCompleteState = this._animateCursor({
-      toValue: 0,
+      toValue: 1,
       duration: this.cursorDuration / 2,
       delay: 100,
```

Comment:
because of the above comment here eventually we need to remove delay completely and make the duration as the full cursorDuration.